ibm/service/database/data_source_ibm_database.go

```diff
@@ -700,11 +720,26 @@ func dataSourceIBMDatabaseInstanceRead(d *schema.ResourceData, meta interface{})
 	}
 	d.Set("auto_scaling", flattenICDAutoScalingGroup(autoSclaingGroup))
 
-	whitelist, err := icdClient.Whitelists().GetWhitelist(icdId)
-	if err != nil {
-		return fmt.Errorf("[ERROR] Error getting database whitelist: %s", err)
+	if _, ok := d.GetOk("whitelist"); ok {
```

Comment:
whitelist and allowlist are computed variables you can't do d.GetOk...

Reply:
can we use cloud v5 API and flatten the result back to both whitelist and allowlist